dhis-2/dhis-support/dhis-support-sql/src/main/java/org/hisp/dhis/db/sql/PostgreSqlBuilder.java

```diff
@@ -249,6 +250,16 @@ public String jsonExtractNested(String column, String... expression) {
     return String.format("%s #>> '{%s}'", column, String.join(", ", expression));
   }
 
+  @Override
+  public String cast(String column, DataType dataType) {
+    return column
+        + switch (dataType) {
+          case NUMERIC -> "::numeric";
```

Comment:
Should we include the `column` argument so that these methods can be called consistently, as the Doris and ClickHouse methods include the column?

```
column + "::numeric";
```

Reply:
The `column` argument is there? `return column...`

Reply:
Sorry my bad.